src/test/run-make-fulldeps/c-link-to-rust-va-list-fn/test.c

```diff
@@ -8,6 +8,7 @@ extern size_t check_list_0(va_list ap);
 extern size_t check_list_1(va_list ap);
 extern size_t check_list_2(va_list ap);
 extern size_t check_list_copy_0(va_list ap);
+extern size_t check_varargs_0(int fixed, ...);
 
```

Comment:
```suggestion
extern size_t check_varargs_1(int fixed, ...)
```

By the way I set up a build on aarch64-linux-gnu to track this [here](https://g.codefresh.io/public/accounts/arm_alpha/pipelines/parched/rust/rust)

Reply:
:+1: that is awesome! Thanks. I've often wondered if I should change the makefile for the test to only run it if the host is one of the OS & architecture combinations I've tested. I've only tested `x86_64` and `i686` on Linux, Mac, and Windows.